main/dapps/dapp-templates.md

```diff
@@ -0,0 +1,56 @@
+## Dapp Templates
+
+When creating a new dapp using `agoric init`, you have the option of
+starting from a number of templates. To use a template other than the default, add the
+`--dapp-template` option:
+
+```sh
+agoric init --dapp-template dapp-fungible-faucet my-fungible-faucet
+```
+
+### Fungible Faucet Dapp
+
+[Github:
+dapp-fungible-faucet](https://github.com/Agoric/dapp-fungible-faucet#fungible-faucet-dapp)
+
+Gives fungible tokens on request. Frontend is raw JavaScript with
+[Material Design Components](https://material-components.github.io/material-components-web-catalog/#/).
+
+### Card Store Dapp
+
+[Github: dapp-card-store](https://github.com/Agoric/dapp-card-store#baseball-card-store-dapp)
+
+Mints NFT baseball cards and allows users to buy them with their
+fungible tokens from the Fungible Faucet Dapp. Frontend is raw
+JavaScript with [Material Design Components](https://material-components.github.io/material-components-web-catalog/#/).
+
+### Encouragement Dapp
+
+[Github:
+dapp-encouragement](https://github.com/Agoric/dapp-encouragement)
+
+Sends messages of encouragement to the user either through the backend
+of the dapp or through minting encouragement NFT tokens in a contract.
+Frontend is raw JavaScript.
```

Comment:
Suggest tightening to "Sends encouragement messages to the user through either the dapp backend or by minting..."